code/messaging/src/phone/java/com/adobe/marketing/mobile/MessagingInternal.java

```diff
@@ -410,11 +410,12 @@ public void error(final ExtensionError extensionError) {
                 // handle the push tracking information from messaging request content event
                 handleTrackingInfo(eventToProcess);
             } else if (MessagingUtils.isEdgePersonalizationDecisionEvent(eventToProcess)) {
```

Comment:
do we have the code in here to ensure this is the event we're looking for?  
in ios we capture the event that requested personalization and verify this is the correct response:
https://github.com/adobe/aepsdk-messaging-ios/blob/inbound-changes/AEPMessaging/Sources/Messaging.swift#L145
https://github.com/adobe/aepsdk-messaging-ios/blob/inbound-changes/AEPMessaging/Sources/Messaging.swift#L163

Reply:
i noticed that this was missing and have added it to the coming pr which processes the XAS payload.

Reply:
i've fixed in the coming pr. I store the event unique identifier to use for comparison later when receiving a edge response event. i'll move those changes into this pr for clarity.